dispatcher/DAQController.py

```diff
@@ -43,6 +43,8 @@ def __init__(self, config, mongo_connector, log, hypervisor):
         self.time_between_commands = int(config['TimeBetweenCommands'])
         self.can_force_stop={k:True for k in detectors}
         self.one_detector_arming = False
+        self.start_cmd_delay = int(config['StartCmdDelay'])
```

Comment:
why int, why not float? 1.5 would not be possible which makes it a little unclear

Reply:
We could